package-res/resources/web/pentaho/type/_doc/spec/ITypeConfigurationRuleSelection.jsdoc

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `spec.ITypeConfigurationRuleSelection` interface describes
+ * the criteria map that determines when a rule is selected for a
+ * given type and context variables.
+ *
+ * It is the type of the [select]{@link pentaho.type.ITypeConfigurationRule#select} attribute
+ * of a type configuration rule.
+ *
+ * Besides the [type]{@link pentaho.type.spec.ITypeConfigurationRuleSelection#type} property,
+ * this interface allows filtering on any of the [context variables]{@link pentaho.spec.IContextVars}.
+ *
+ * @name pentaho.type.spec.ITypeConfigurationRuleSelection
+ * @interface
+ *
+ * @see pentaho.type.spec.ITypeConfigurationRule
+ */
+
+/**
+ * The id or [ids]{@link pentaho.type.Type#id} of the value types that the rule applies to.
+ *
+ * When {@link Nully} or unspecified, it defaults to the root value type,
+ * [value]{@link pentaho.type.Value}.
+ *
+ * The specified id or ids may be standard
+ * [short ids]{@link pentaho.type.Type#shortId} or absolute.
+ * When an id does not contain any "/" character, it is considered a standard type, and, thus,
+ * is taken relative to Pentaho's `"pentaho/type"` module.
+ * Otherwise, it's considered absolute.
```

Comment:
Suggested wording:
When an id does not contain any "/" character, it is considered a standard type and is taken relative to Pentaho's `"pentaho/type"` module. Otherwise, it is considered absolute.